internal/db/db_blueprints.go

```diff
@@ -159,15 +172,47 @@ func (db *dB) DeleteBlueprint(id uuid.UUID, orgID, accountNumber string) error {
 	return err
 }
 
-func (db *dB) GetBlueprints(orgID, accountNumber string, limit, offset int) ([]BlueprintWithNoBody, int, error) {
+func (db *dB) FindBlueprints(orgID, search string, limit, offset int) ([]BlueprintWithNoBody, int, error) {
+	ctx := context.Background()
+	conn, err := db.Pool.Acquire(ctx)
+	if err != nil {
+		return nil, 0, err
+	}
+	defer conn.Release()
+	searchQuery := fmt.Sprintf("%s%s%s", "%", search, "%")
```

Comment:
I still think this is too messy for no reason, how about:

```suggestion
	searchQuery := "%" + search + "%"
```